main.tf

```diff
@@ -50,6 +50,10 @@ resource "kubernetes_deployment" "deployment" {
             name  = "SCIMMA_ADMIN_PROD"
             value = "True"
           }
+          env {
+            name  = "KAFKA_USER_AUTH_GROUP"
+            value = "$(var.user_auth_group)"
+          }
```

Comment:
Hm, I think `SCIMMA_ADMIN_PROD` was actually a mistake that we shouldn't replicate. The intent of this module is to provide an independent way to stand up an HTTP service inside Kubernetes; we'd be able to run any number of totally different HTTP services using this. It really ought not to be too specific to the admin application.

I think it makes more sense for this module to accept a list of env key-value pairs as an input. The terraform required to do that is somewhat sophisticated - I think it would require a [dynamic](https://www.terraform.io/docs/language/expressions/dynamic-blocks.html) block:
```hcl
# vars.tf

variable "env_vars" {
  type = list(object({
    name  = string
    value = string
  }))
  default = []
  description = "Environment variables to provide to the application container"""
}
```

```hcl
# main.tf
dynamic "env" {
  for_each = var.env_vars
  iterator = "x"
  content {
    name  = x.name
    value = x.value
  }
}
```

I think that makes more sense for making this reusable, at least.